securitycenter/api/src/NotificationReceiver.g.cs

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START scc_receive_notifications]
+using System.Collections.Generic;
+using Google.Cloud.PubSub.V1;
+using Google.Protobuf;
+using Grpc.Core;
+using System;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class NotificationReceiver
```

Comment:
Is this a Security Center sample or a Pub Sub sample?
We have Pub Sub samples that do this or very similar. Consider reusing one of those by simply adding this sample's tag there.

Reply:
Yeah, I think reusing an existing sample is the way. If so, I think this file won't be necessary anymore.